src/src/WebServer/Markup.cpp

```diff
@@ -118,6 +118,7 @@ void addSelector_options(int optionCount, const __FlashStringHelper * options[],
       attr_str = attr[x];
     }
     addSelector_Item(options[x], index, selectedIndex == index, false, attr_str);
+    if (x % 10 == 0) delay(0);
```

Comment:
I did change some of the pin selectors (`addFormPinSelect` etc) , to no longer create an array of choices.
This way the data is immediately flushed to the TXBuffer.
Maybe this should also be done here, if you exceed 10 elements?

Reply:
I build a list of all DecodeType options available (97 atm), and offer these at bot the normal `Code` and `Alt. Code` fields, and there are 10 of each. Not sure if the added code would weight more in this case. And it keeps the code a bit cleaner IMHO. That `delay(0)` call seems beneficiary to every option list, it sure made the UI more stable when opening this new page.